ckan/logic/action/get.py

```diff
@@ -518,10 +519,13 @@ def organization_list(context, data_dict):
     :rtype: list of strings
 
     '''
-    _check_access('organization_list', context, data_dict)
-    data_dict['groups'] = data_dict.pop('organizations', [])
-    data_dict.setdefault('type', 'organization')
-    return _group_or_org_list(context, data_dict, is_org=True)
+    try:
+        _check_access('organization_list', context, data_dict)
+        data_dict['groups'] = data_dict.pop('organizations', [])
+        data_dict.setdefault('type', 'organization')
+        return _group_or_org_list(context, data_dict, is_org=True)
+    except logic.NotAuthorized:
+        base.abort(403, _('You are not authorized to see a list of organizations'))
```

Comment:
aborting is something controllers do. I think it's correct for this logic function to just raise the NotAuthorized exception from the _check_access call. The problem is one of the controllers that calls this action isn't handling NotAuthorized.